recipes/util-linux-libuuid/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.3.0":
```

Comment:
Is this version number really meaningfull ? It did not change in two years https://github.com/util-linux/util-linux/blame/master/meson.build#L11 in spite of several tags on the repo

Reply:
Yeah, fair enough. I saw this in the meson.build file and assumed that it was well managed, and liked that it referred to the libuuid version directly rather than the bundle. It doesn't seem tenable to keep 1.3.0 though, so I'll update it to use the util-linux version